sway/desktop/fx_renderer/fx_framebuffer.c

```diff
@@ -5,7 +5,17 @@ void fx_framebuffer_bind(struct fx_framebuffer *buffer) {
 	glBindFramebuffer(GL_FRAMEBUFFER, buffer->fb);
 }
 
-void fx_framebuffer_create(struct fx_framebuffer *buffer, int width, int height, bool bind) {
+void fx_framebuffer_init(struct fx_framebuffer *buffer) {
```

Comment:
Let's have this return a newly created fx_framebuffer, and rename is to fx_framebuffer_create. We can also do any other initial work here. We can then update the existing `fx_framebuffer_create`, to be `fx_framebuffer_update`.